### ISSUE TITLE
 Can not figure out sign config, please make sure you have only one sign config in your build.gradle, or set signConfigName in okbuck dsl.

### ISSUE THREAD

**Issue by 2kpurple** [Piasy/OkBuck#56]
我是按README_ZH来搞的。
最后是这个错误 >_> 怎么破

org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':okbuck'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:69)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.PostExecutionAnalysisTaskExecuter.execute(PostExecutionAnalysisTaskExecuter.java:35)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:64)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:42)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:53)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.AbstractTask.executeWithoutThrowingTaskFailure(AbstractTask.java:310)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.executeTask(AbstractTaskPlanExecutor.java:79)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.processTask(AbstractTaskPlanExecutor.java:63)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.run(AbstractTaskPlanExecutor.java:51)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: java.lang.IllegalStateException: Can not figure out sign config, please make sure you have only one sign config in your build.gradle, or set signConfigName in okbuck dsl.
        at com.github.piasy.okbuck.helper.ProjectHelper.createKeystoreRule(ProjectHelper.groovy:443)
        at com.github.piasy.okbuck.helper.ProjectHelper$createKeystoreRule$6.call(Unknown Source)
        at com.github.piasy.okbuck.dependency.DependencyProcessor.processKeystore(DependencyProcessor.groovy:68)
        at com.github.piasy.okbuck.dependency.DependencyProcessor.process(DependencyProcessor.groovy:59)
        at com.github.piasy.okbuck.dependency.DependencyProcessor$process.call(Unknown Source)
        at com.github.piasy.okbuck.generator.XBuckFileGenerator.generate(XBuckFileGenerator.groovy:58)
        at com.github.piasy.okbuck.generator.XBuckFileGenerator$generate.call(Unknown Source)
        at com.github.piasy.okbuck.OkBuckGradlePlugin.applyWithoutBuildVariant(OkBuckGradlePlugin.groovy:135)
        at com.github.piasy.okbuck.OkBuckGradlePlugin$_apply_closure3.doCall(OkBuckGradlePlugin.groovy:74)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:558)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:539)
        at org.gradle.api.internal.tasks.TaskMutator$1.execute(TaskMutator.java:77)
        at org.gradle.api.internal.tasks.TaskMutator$1.execute(TaskMutator.java:73)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
        ... 14 more

**Comment by Piasy** [comment]
@2kpurple 
```java
Can not figure out sign config, please make sure you have only one sign config in your buildgradle, or set signConfigName in okbuck dsl
```

报错提示这个，你需要指定一下签名配置，可以参考一下[demo的build.gradle](https://github.com/Piasy/OkBuck/blob/master/build.gradle)

**Comment by 2kpurple** [comment]
@Piasy 好的~我试试看！谢谢！ :)))

**Comment by 2kpurple** [comment]
@Piasy 如果是多工程依赖的话要怎么做呢？

现在是报这个了 resPackage key-value pair must be set for sub project

**Comment by Piasy** [comment]
@2kpurple [demo的build.gradle](https://github.com/Piasy/OkBuck/blob/master/build.gradle)里面也有为多个依赖Module配置资源文件包名。

**Comment by 2kpurple** [comment]
@Piasy 看到了 实在抱歉 >_> 都是在写完编译的时候等得久。。然后顺便看看。。

**Comment by Piasy** [comment]
@2kpurple 没关系，其实没必要等android studio sync完毕。直接修改之后在命令行里面运行`./gradlew okbuck`即可

**Comment by 2kpurple** [comment]
@Piasy 是不是不支持windows >_>

**Comment by Piasy** [comment]
@2kpurple 是的目前官方没有支持。

**Comment by 2kpurple** [comment]
@Piasy 今晚换个ubuntu试试...编译过了, 根目录运行buck, 说找不到buck

**Comment by Piasy** [comment]
fixed by v0.4.10
